cli/.npmignore

```diff
@@ -0,0 +1 @@
+bin/cargo-generate
```

Comment:
Can we just do `bin/*`?

Reply:
actually you are right we should load everything by release tag the way it is right now this is a setup for disaster because by including `bin/*` we include latest build instead of releases of all programs and prover binaries

Reply:
I would like to do that in a separate pr though